tests/local_mode/test_single_label_classifier_local.py

```diff
@@ -0,0 +1,101 @@
+import pytest
```

Comment:
Actually, local vs. remote mode in tests will be deprecated in the future. The distinction is not always clear. So please transfer those test cases to the remote mode. And make sure to always test the case when you call `load` to use a deployed pipeline.

Also (in a separate PR), please test the cases when the pipeline is deployed on `bob` for example but the string is on `alice`, and the classifier has access right only to 'bob', and then access {'*'}